dspace-server-webapp/src/test/java/org/dspace/app/rest/DiscoveryRestControllerIT.java

```diff
@@ -4960,4 +4960,141 @@ public void discoverSearchObjectsTestForAdministrativeViewWithFilters() throws E
                 ))
                 .andExpect(jsonPath("$._links.self.href", containsString("/api/discover/search/objects")));
     }
+
```

Comment:
Since @artlowel mentioned that the issue could occur with more than just the configuration parameter, could you provide a test in which all of the params for the endpoint (listed here `org.dspace.app.rest.DiscoveryRestController#getFacetValues`) are tested and verified? This way we can be sure that we haven't missed any

Reply:
test to verify the other parameters have been added